File: bee/simples/2028.go
package main

import (
	"fmt"
	"io"
)

func seq(num int, cont int) {
	lista := []int{0}
	for i := 1; i <= num; i++ {
		for j := 0; j < i; j++ {
			lista = append(lista, i)
		}
	}
	if len(lista) == 1 {
		fmt.Printf("Caso %d: %d numero\n", cont, len(lista))
	} else {
		fmt.Printf("Caso %d: %d numeros\n", cont, len(lista))
	}

	for i := 0; i < len(lista); i++ {
		if i == len(lista)-1 {
			fmt.Println(lista[i]) // Último item sem espaço
		} else {
			fmt.Print(lista[i], " ") // Espaço entre os números
		}
	}
}

func main() {
	cont := 1
	for {
		var o int
		_, err := fmt.Scanf("%d", &o)

		if err == io.EOF {
			break
		}
		seq(o, cont)
		cont++
		fmt.Println()
	}
}

File: bee/simples/2028.py
def seq(num: int, cont: int):
    lista = [0]
    for i in range(num + 1):
        for _ in range(i):
            lista.append(i)

    print(f"Caso {cont}: {len(lista)}", "numero" if len(lista) == 1 else "numeros")
    for i in range(len(lista)):
        if i == len(lista) - 1:  # Verifica se é o último item
            print(lista[i])
        else:
            print(lista[i], end=" ")



cont = 1
while True:
    try:
        o = int(input())
        seq(o, cont)
        print()
        cont += 1
    except EOFError:
        break
